scripts/Dockerfile

```diff
@@ -13,16 +18,45 @@ RUN add-apt-repository ppa:rock-core/qt4 -y && \
 RUN rm -rf /catkin_ws && \
     mkdir -p /catkin_ws/src && \
     cd /catkin_ws && \
-    catkin init
+    catkin init && \
+    cd /catkin_ws/src && \
+    git clone https://github.com/RUFFY-369/CrazyS.git && \
+    git clone https://github.com/RUFFY-369/mav_comm.git && \
+    cd /catkin_ws
 COPY .rosinstall /tmp/
 RUN vcs import /catkin_ws/src  < /tmp/.rosinstall
 
+
+#CrazyS
+RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
```

Comment:
Collide from line 31 to 37 in only one RUN command to reduce image layers

Reply:
done the changes in the cleaned Dockerfile